docs/src/prototypes/mentions/MentionsDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import { Dropdown, DropdownItemProps, Provider } from '@stardust-ui/react'
+
+import { MentionsContainerProps } from './MentionsEditor'
+
+const MentionsDropdown: React.FunctionComponent<MentionsContainerProps> = props => {
+  const { searchQuery, items, onOpenChange, onSearchQueryChange, onInputKeyDown } = props
+
+  const renderItem = React.useCallback(
+    (Item: React.ReactType, props: DropdownItemProps, boldFont: number) => {
+      const { header = '', ...rest } = props
+      const headerStr = String(header)
+      const queryStartIndex = headerStr.indexOf(searchQuery)
+
+      if (queryStartIndex < 0) return <Item {...props} />
+      const queryEndIndex = queryStartIndex + searchQuery.length
+
+      return (
+        <Item
+          {...rest}
+          header={{
+            content: (
+              <span>
+                {headerStr.substring(0, queryStartIndex)}
+                <span style={{ fontWeight: boldFont }}>
+                  {headerStr.substring(queryStartIndex, queryEndIndex)}
+                </span>
+                {headerStr.substring(queryEndIndex)}
+              </span>
+            ),
+          }}
+        />
+      )
+    },
+    [searchQuery],
+  )
+
+  return (
+    <Provider.Consumer
```

Comment:
As we require `react@16.8` I suggest to reexport this context as hook:
```
const siteVariables = useStardustTheme()
```

Reply:
agreed, do we have any established pattern on where to create hooks?
where would you place `useStardustTheme`?

also, I'd say this should be a separate PR that will address all usages of `Provider.Consumer` in our codebase. @layershifter what do you think?

Reply:
Yep, let's do it in a separate PR